BrickKit/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.2-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.4.1-all.zip
```

Comment:
can't use later version of gradle?  No Jacoco compatibility issue in this lib...(why we're stuck on 5.4.1 in the app IIRC)

Reply:
5.6.2 worked!